utils/forest_utils/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2021"
 anyhow.workspace           = true
 const_format               = "0.2.26"
 digest                     = "0.10.5"
-isahc.workspace            = true
+isahc                      = { version = "1.7", default-features = false }
```

Comment:
could we instead of re-defining the version have `workspace => true, default-features => false` ? Or am I missing something?

Reply:
Ah, that makes sense. Thanks for clarifying.